detect_secrets/filters/heuristic.py

```diff
@@ -165,6 +165,8 @@ def is_indirect_reference(line: str) -> bool:
 
         secret = request.headers['apikey']
     """
+    if len(line) > 1000:
```

Comment:
@jpdakran : we should document design rationales in code to assist future developers in understanding the codebase.

Reply:
Understood. Please see https://github.com/Yelp/detect-secrets/pull/516 for update.